src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -202,6 +150,33 @@ def main(self):
             default=[],
             help='Devices to provision OSDs',
         )
+        parser.add_argument(
+            '--db-devices',
+            nargs='*',
+            type=arg_validators.ValidDevice(),
+            default=[],
+            help='Devices to provision OSDs db volumes',
+        )
+        parser.add_argument(
+            '--wal-devices',
+            nargs='*',
+            type=arg_validators.ValidDevice(),
+            default=[],
+            help='Devices to provision OSDs wal volumes',
+        )
+        parser.add_argument(
+            '--journal-devices',
+            nargs='*',
+            type=arg_validators.ValidDevice(),
+            default=[],
+            help='Devices to provision OSDs journal volumes',
+        )
+        parser.add_argument(
+            '--no-auto',
+            action='store_true',
+            help=('deploy standalone OSDs if rotational and non-rotational drives'
```

Comment:
This is not going to hold true for filestore. Saying `standalone` also reads a bit odd, I don't think we refer to a bluestore OSD with only block (data) defined as standalone. Maybe it would be better to describe that this flag would disable the automatic grouping of devices by checking solid and rotational?

Reply:
> This is not going to hold true for filestore.

Why not? What if a user wants to deploy standalone (for lack of a better term) filestore OSDs on spinners and SSDs?

> Saying `standalone` also reads a bit odd, I don't think we refer to a bluestore OSD with only block (data) defined as standalone. Maybe it would be better to describe that this flag would disable the automatic grouping of devices by checking solid and rotational?

I'm aware of the lack of standard terms here. I'm happy to make this a bit more verbose and clear.

Reply:
But what if a user passes a mix of HDD's and SSD's in the devices list but intends to only deploy standalone OSD's (OSDs occupying only one device)? Without `--no-auto` or similar this is not possible.

Reply:
If they have a mix of HDD'S and SSD's then it'd use a MixedType strategy and would not collocate all services on the same device. So this ``--no-auto`` flag is an attempt to always make OSDS with everything collocated on the same device regardless of the type of device? If so, maybe naming it ``--collocated`` makes more sense? Collocated is the term ceph-ansible used to describe an OSD with either block.db or the journal on the same device as data.

Reply:
I'm happy to rename the option, but it is needed in any case.

Ideally we'd have some project wide term to use for the different OSD deployment modes.